sysutils/pyenv/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            pyenv pyenv 2.3.2 v
+revision                0
+
+categories              sysutils python devel
+license                 MIT
+maintainers             nomaintainer
+supported_archs         noarch
+
+description             Simple Python Version Management
+long_description        pyenv lets you easily switch between multiple versions of Python.\
+                        It is simple, unobtrusive, and follows the UNIX tradition of\
+                        single-purpose tools that do one thing well.
+
+github.tarball_from     archive
+checksums               rmd160 28cb721fcd3d3fbdb95d44708e5d773e7bacab01 \
+                        sha256 150ac8f7161c00e8e613bf5e273306f674b60037e3dace9c6bb7611dceb17144 \
+                        size   713692
+
+depends_lib-append \
+                        port:openssl11 \
+                        port:readline \
+                        port:pkgconfig \
+                        port:autoconf
+
```

Comment:
Prereqs included here @thecesrom (this section lists them as dependencies).

Reply:
I think the CPP and LD flags are also set with default values by MacPorts itself (again, remembering back based on when I originally pieced this together from the pyenv setup guides).

Reply:
I'll go ahead and create a local repo and share feedback. Thanks, @gleeblezoid!

Reply:
Yup, the defaults are listed in https://guide.macports.org/#reference.phases.configure:

- `LDFLAGS` `Default: -L${prefix}/lib -Wl,-headerpad_max_install_names`
- `CPPFLAGS` `Default: -I${prefix}/include`

Reply:
I'm still happy to re-test the install and setup on a wiped machine with no other ports but hopefully the above helps with the questions re prereq installs and the flags setup.

Reply:
I recall some of this being pretty fiddly way back when, so very grateful for any input you've got @thecesrom